### PR TITLE
fix(Argus): fix issue regarding catch failed runs

### PR DESCRIPTION
**Pull request by temichus**
Since we are now creating argus run even before sct runner is created we need to use new argus client enpoind to post sct runner infor when it is created

set typing-extensions==4.5.0 due to bug https://github.com/pydantic/pydantic/pull/5826 in pydantic==1.8.2

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Review by k0machi — APPROVED** [review]

File: sdcm/tester.py [hunk 1]
```diff
@@ -395,11 +395,15 @@ def init_argus_run(self):
...
                     commit_id=get_git_commit_id(),
-                    runner_public_ip=get_sct_runner_ip(),
-                    runner_private_ip=get_my_ip(),
                     sct_config=self.params,
                 )
                 self.log.info("Initialized Argus TestRun with test id %s", self.test_config.argus_client().run_id)
+            self.test_config.argus_client().set_sct_runner(
+                public_ip=get_sct_runner_ip(),
+                private_ip=get_my_ip(),
+                region="undefined_region",
```
Comment (juliayakovlev) [inline sdcm/tester.py:404]: Is it impossible to get runner region?

**Review by juliayakovlev — COMMENTED** [review]
looks ok. one question

File: sdcm/tester.py (hunk 1 above)
Comment (juliayakovlev) [reply sdcm/tester.py:404]: ok

**Review by juliayakovlev — APPROVED** [review]

File: sdcm/tester.py (hunk 1 above)
Comment (fruch) [reply sdcm/tester.py:404]: > According to discussion with @k0machi we are not showing region of sct runner in Agrus now. Will create task to determine an post runner region with low priority. 

Runner is always in the first region from the test configuration.

File: requirements.in [hunk 2]
```diff
@@ -52,3 +52,4 @@ questionary==1.10.0
 pbr==1.4
 hdrhistogram==0.9.2
 deepdiff==6.2.3
+typing-extensions==4.5.0
```
Comment (fruch) [inline requirements.in:55]: Why are we pinning this one and not upgrading pydantic?

**Pull request merged**